client/js/socket-events/more.js

```diff
@@ -31,7 +31,13 @@ socket.on("more", function(data) {
 
 	// restore scroll position
 	const position = chan.height() - heightOld;
-	scrollable.scrollTop(position);
+	scrollable.finish().scrollTop(position);
```

Comment:
`finish` is here for the same reason we have it when using page keys. As we have no clean way of updating final scroll position while jQuery is animating, we have to stop the animation.